README.rst

```diff
@@ -42,12 +42,14 @@ Installation
 
     pip install dwave-system[drivers] --extra-index-url https://pypi.dwavesys.com/simple
 
-Downloading this package with drivers installs a dependency called dwave-system-tuning that has a restricted license. To view the license details:
+.. note::
+    The installation steps have changed since v0.3.0. The D-Wave drivers are now an optional but recommanded dependency. 
```

Comment:
The install instructions changed since 0.2.x. Since version 0.3.0 ...

Reply:
Have changed as of version 0.3.0?